src/Sulu/Bundle/MediaBundle/Resources/config/services.xml

```diff
@@ -171,5 +171,23 @@
         <service id="sulu_media.security_context" class="Sulu\Component\Security\Authorization\SecurityCondition">
             <argument>sulu.media.collections</argument>
         </service>
+
+        <service id="sulu_media.system_collections.cache" class="Sulu\Component\Cache\ConfigCache">
```

Comment:
Why have you defined that as a service? Without that it would not be necessary to have our own `ConfigCache` class...